api/openapi.gen.yaml

```diff
@@ -173,6 +175,8 @@ components:
       properties:
         body:
           type: string
+        fingerprint:
+          type: string
         id:
           format: int64
           type: integer
```

Comment:
QA: We are adding new field "fingerprint" but it is read-only, if you try to set it to any value it will be ignored.

Also the description describes that pubkeys now must be unique: both name and pubkey itself (the public key body). If it's not, error 422 is returned.